docs/javascript/browser-apis-and-jquery.md

```diff
@@ -355,4 +355,52 @@ You can select multiple elements as well. For example, elements that all share t
 <figure markdown>
 ![jquery get elements example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/get-elements-by-class-jquery.png?raw=true){ width="100%" }
 <figcaption></figcaption>
-</figure>
\ No newline at end of file
+</figure>
+
+//what is needed?
+Styling DOM Nodes
+Events
+Timers
+### Styling DOM Nodes 
+
+### Events
+
+### Timers 
+Javascript has two `time intervals` methods that allows us to execute code at a given time. 
+
+The `setInterval` function will execute a given function after a waiting a certain amount of time. 
+
+``` 
+setInterval(function, ms)
+```
+
+```html
```

Comment:
Use this https://docs.codewizardshq.com/javascript/browser-apis-and-jquery/#using-a-script-tag-in-the-body as a general template for HTML.